guides/common/modules/proc_executing-a-remote-job.adoc

```diff
@@ -40,11 +40,11 @@ You can also define a one-time future job, or set up a recurring job.
 For recurring tasks, you can define start and end dates, number and frequency of runs.
 You can also use cron syntax to define repetition.
 ifndef::orcharhino[]
-For more information about cron, see the https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/system_administrators_guide/ch-automating_system_tasks[Automating System Tasks] section of the {RHEL} 7 _System Administrator's Guide_.
+For more information about cron, see https://www.redhat.com/sysadmin/automate-linux-tasks-cron[Automate your Linux system tasks with cron].
 endif::[]
 
 . Click *Submit*.
-This displays the *Job Overview* page, and when the job completes, also displays the status of the job.
+You can view status of the jobs in the *Recent Jobs* section on the same page.
```

Comment:
The previous description says that clicking submit displays the new page. If this is still the case, then it wouldn't make sense to say "You can view status of the jobs on the **same** page." I would personally replace **same** with **displayed**.

Reply:
I confirmed in the UI of Satellite in the demo video. The status of the job is viewed on the same page. It will not load a dedicated page.

Reply:
Understood :+1: